.byebug_history

```diff
@@ -0,0 +1,20 @@
+exit
```

Comment:
At our next meeting let's go over making sure files like these don't get committed. This file is a log of everything you've typed in byebug, and could contain sensitive info. Additionally, you don't want to overwrite this file in other people's projects when they check out the code.